README.md

```diff
@@ -59,21 +59,34 @@ This meta-role requires a permission policy similar to the following:
 | Name | Version |
 |------|---------|
 | terraform | ~> 0.12.0 |
-| aws | ~> 3.0 |
+| aws | ~> 3.38 |
 
 ## Providers ##
 
 | Name | Version |
 |------|---------|
-| aws | ~> 3.0 |
+| aws | ~> 3.38 |
+
+## Modules ##
+
+No modules.
+
+## Resources ##
+
+| Name | Type |
+|------|------|
+| [aws_iam_role.the_role](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/iam_role) | resource |
+| [aws_iam_role_policy.cert_policy](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/iam_role_policy) | resource |
+| [aws_iam_policy_document.assume_role_doc](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/iam_policy_document) | data source |
+| [aws_iam_policy_document.cert_doc](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/iam_policy_document) | data source |
 
 ## Inputs ##
 
 | Name | Description | Type | Default | Required |
 |------|-------------|------|---------|:--------:|
-| account_ids | AWS account IDs that are to be allowed to assume the role | `list(string)` | `[]` | no |
-| cert_bucket_name | The name of the AWS S3 bucket where certificates are stored | `any` | n/a | yes |
-| cert_path | The path to the certificates in the AWS S3 bucket.  For example, the certificate files for site.example.com are expected to live at <cert_path>/site.example.com/* | `string` | `live` | no |
+| account\_ids | AWS account IDs that are to be allowed to assume the role | `list(string)` | `[]` | no |
+| cert\_bucket\_name | The name of the AWS S3 bucket where certificates are stored | `any` | n/a | yes |
+| cert\_path | The path to the certificates in the AWS S3 bucket.  For example, the certificate files for site.example.com are expected to live at <cert\_path>/site.example.com/* | `string` | `"live"` | no |
```

Comment:
I'm not sure if this is actually useful or if it just confuses the description. I leave it to your judgement.

```suggestion
| cert\_path | The path to the certificates in the AWS S3 bucket.  For example, the certificate files for site.example.com are expected to live at "<cert\_path>/site.example.com/*". | `string` | `"live"` | no |
```

Reply:
See commit 62847c1.  I don't think the additional quotes add anything so I omitted them.